template-compose/data/src/test/java/co/nimblehq/template/compose/data/test/MockUtil.kt

```diff
@@ -29,4 +29,8 @@ object MockUtil {
     val errorResponse = ErrorResponse(
         message = "message"
     )
+
+    val responses = listOf(
+        co.nimblehq.template.compose.data.response.Response(id = 1)
```

Comment:
Optional: We can import like `import co.nimblehq.sample.compose.data.response.*` or using `as`